### DISCUSSION TITLE
Anzeigevariante funktioniert nicht

### DISCUSSION THREAD

**Discussion by duffy6** [Zefau/ioBroker.jarvis#2086] [Q&A] [answered]
Hallo zusammen,

irgendwas scheine ich falsch zu machen mit der Anzeigevariante.
ich möchte alle Temperaturwerte unter 0°C rot einfärben.

```
{
  ">0": {
    "color": "00ac00"
  },
  "<0": {
    "color": "#f00"
  }
}
```

Es geschieht NICHTS :-(

![1](https://user-images.githubusercontent.com/30000750/205346338-bbc0a6f1-f203-4427-a617-7a63958e3cbd.PNG)
![2](https://user-images.githubusercontent.com/30000750/205346553-65a7bc07-6117-4c65-aeaf-717732582deb.PNG)

Jarvis V3.1.2

**Comment by Zefau** [comment]
Rot ist kleiner 0, aber deine Werte sind größer als 0. Die Bedingung ist also nicht erfüllt. 

Bei größer 0 hast du die Raute # beim Hex Code vergessen.

**Comment by duffy6** [comment, marked as answer]
Danke!
Aber es funktioniert leider trotzdem nicht:

![1](https://user-images.githubusercontent.com/30000750/205404520-8c306695-6bac-4d51-9476-f1f4fe82d4c6.PNG)
![2](https://user-images.githubusercontent.com/30000750/205404533-9b1f06f3-5969-4002-bc08-e48a58525558.PNG)

**Reply by Zefau** [reply]
Schau mal ins Wiki und ich mach bei Gelegenheit mal ein Video zur Anleitung.

Du hast den Style in der Anzeige Variante, aber die muss in den Body oder Datenpunkt Stil.

Die Anzeige Variante mappt nur Daten. Also wenn du `true` hast und dafür einen Wert haben willst.

**Comment by duffy6** [comment]
Super, dankeschön!
